tests/Collections/PrefixTrieTests.cs

```diff
@@ -112,16 +109,13 @@ public void TestAllKeys()
 
             var withPrefixO = trie.AllKeys("o").ToList();
 
-            Assert.Equal(1, withPrefixO.Count);
-            Assert.Equal("one", withPrefixO[0]);
+            withPrefixO.Should().HaveCount(1);
```

Comment:
```suggestion
            withPrefixO.Should().ContainSingle().Which.Should().Be("one");
```

It combines both the conditions in one line. Alternatively:

```suggestion
            withPrefixO.Should().ContainSingle(elmt => elmt == "one");
```

Or

```suggestion
            // the array wrapper is unnecessary, but more explicit
            withPrefixO.Should().BeEquivalentTo(new [] { "one" });
```